joomla/joomla-cms
# Reverse ordering

# Executive summary

This adds a additional ordering option to the category ordering. 
## Background information

A while ago we changed the default for the ordering field, before that change a new article got „0“ as ordering so that article was the first one in a blog view when ordered by ordering. The change was that a new article now gets max(ordering) as default value and that will show an article as the last when ordered by ordering. The change was good because we had some performance problems when the ordering value was updated on a site with many articles. So no complaining about the change itself. The problem now is that you have to reorder articles because your article is now the last one in the category. If you do not have this manual step then you need to switch the ordering and use reverse ordering.  
# Backwards compatibility

Because it is a new option nothing should change
# Translation impact

Two new language tags added.
# Core changes

Addition to ContentHelperQuery::orderbySecondary
# Testing instruction
- Apply patch
- You need a category view with articles
- Set the the secondary ordering in Articles options to     „Ordering Reverse“
- Set the the secondary ordering for the menu to „Use Global“
- check the view and compare with the backend list of articles, use in both occasions the same ordering.

Should be the same ordering in front and backend
- you can also change the menu item ordering and play around with the combinations
